playground/nextjs/pages/mobile-app-attribution.tsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect } from 'react'
+import { posthog } from '@/src/posthog'
+
+const MobileAppAttribution = () => {
+    useEffect(() => {
+        if (
+            navigator.userAgent.toLowerCase().includes('linkedin') ||
+            navigator.userAgent.toLowerCase().includes('facebook')
+        ) {
+            const params = new URLSearchParams(window.location.search)
+            params.set('__ph_distinct_id', posthog.get_distinct_id())
+            params.set('__ph_is_identified', posthog._isIdentified() ? 'true' : 'false')
```

Comment:
We should remove the _ from this before telling people to use it, we strongly discourage people from using function that begin with _ as they are not part of the public interface and might change